common-crafting.lic

```diff
@@ -12,26 +12,26 @@ module DRCC
     ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
   end
 
-  def find_empty_crucible
+  def find_empty_crucible(idle_room = nil, crucibles)
```

Comment:
This:

def foo(hometown = get_settings["hometown"])

Could be done a couple different ways as long as it can be called with
defaults or be overridden.

On Tue, Sep 6, 2016 at 1:57 PM, Robert Herbig <notifications@github.com>
wrote:

> In common-crafting.lic
> <https://github.com/rpherbig/dr-scripts/pull/751#discussion_r77686257>:
>
> > @@ -12,26 +12,26 @@ module DRCC
> >      ['nothing in there'].include?(DRC.bput('look in cruc', 'nothing in there', 'you see'))
> >    end
> >
> > -  def find_empty_crucible
> > +  def find_empty_crucible(idle_room = nil, crucibles)
>
> I'm not sure what you mean. Something like def foo(hometown='Crossing')?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/rpherbig/dr-scripts/pull/751/files/d56664e67f7772e62511263960f200f4536e4317#r77686257>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ARQKjuewAUd9anoX3RUmfa8lNLXhUy0Sks5qnamZgaJpZM4J2CdL>
> .
>